SDL/Event.h

```diff
@@ -84,10 +84,37 @@ namespace SDL {
     int* superbinCPU;
     int8_t* pixelTypeCPU;
 
+    // Stuff that used to be global
+    uint16_t nModules;
+    uint16_t nLowerModules;
+    std::shared_ptr<modulesBuffer<Dev>> modulesBuffers;
+    std::shared_ptr<pixelMap> pixelMapping;
+    std::shared_ptr<EndcapGeometry<Dev>> endcapGeometry;
+    std::shared_ptr<TiltedGeometry<Dev>> tiltedGeometry;
+    std::shared_ptr<ModuleConnectionMap<Dev>> moduleConnectionMap;
```

Comment:
can this all be const?
This is required for cases where data can be used directly from the event setup; otherwise we'd have to make a copy somewhere, which is not good

Reply:
Okay, I can adjust things to make them const. It will be easier once I make the change I mentioned above.